project-koku/koku
# [COST-1232] Instance-Type GroupBy Fix

Fixes group by requests for Instance-type on AWS, GCP, OpenShift on cloud. https://issues.redhat.com/browse/COST-1232


Testing ~
1. Load test data.
2. Navigate to our first test page on AWS with the link http://localhost:8000/api/cost-management/v1/reports/aws/instance-types/?group_by[instance_type]=* . Once loaded, it will show all available instance types from the data you have loaded. Make sure to scroll to check that more than one has loaded, pick the first one and group by that, replacing the * with the name of the instance type.
3. Search for that instance type, EX http://localhost:8000/api/cost-management/v1/reports/aws/instance-types/?group_by[instance_type]=m5.large, once the page is loaded make sure that only that instance type was loaded and grouped by, you can do this by scrolling all the way down the page and checking the instance-type field.  
4. Repeat steps 2-3 for GCP, and OpenShift on cloud.
